### PR TITLE
Issue #412 - Add warning about Kibana Dev Console

### PR DESCRIPTION
**Pull request by DonNateR**
Fixes issue #412

### CODE REVIEW

File: docs/siem-apis.asciidoc [hunk 1]
```diff
@@ -2,6 +2,8 @@
 [[security-apis]]
 = Elastic Security APIs
 
+WARNING: Any request or response examples created in {kib} Dev Tool Console cannot be used to create an API request.
```
Comment (jmikell821) [inline docs/siem-apis.asciidoc:5]: @DonNateR I don't think this is what we're trying to say. You _can_ make API requests via the dev tool console, you just can't copy and paste the examples; you have to type them in manually, I think. @louisong @spong is this correct?

Comment (DonNateR) [reply docs/siem-apis.asciidoc:5]: My impression from reading the issue is that the call itself wouldn't work, even if the responses in the Dev Console were valid.

Comment (jmikell821) [reply docs/siem-apis.asciidoc:5]: Gotcha - thanks. Slight suggestion.

```suggestion
WARNING: Do not use the {kib} Dev Tool Console to make any {es-sec} API requests. 
```

Comment (spong) [reply docs/siem-apis.asciidoc]: Yeah specifically here, Kibana API requests cannot be made via the dev tools. This is the message used on the kibana docs here: https://www.elastic.co/guide/en/kibana/7.10/using-api.html. Perhaps we should just use that message as well?

image

**Review by spong — APPROVED** [review]
LGTM! Thanks @DonNateR! 🙂

File: docs/siem-apis.asciidoc [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 You can use these APIs to interface with {es-sec} features:
 
+NOTE: The {kib} Console supports only Elasticsearch APIs. You are unable to interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```
Comment (jmikell821) [inline docs/siem-apis.asciidoc:7]: @DonNateR slight change for conciseness, then OK to merge. 

```suggestion
NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

**Review by jmikell821 — APPROVED** [review]
OK to merge.

**Pull request merged**